package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@cerebroapp/cerebro-mac-apps",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

As you introduced the `refresh` command, we should publish a minor. 

```suggestion
  "version": "1.1.0",
```